src/meltano/core/utils/__init__.py

```diff
@@ -432,5 +432,25 @@ def hash_sha256(value: str) -> str:
 
     Returns:
         The hashed value of the given string.
+
+    Raises:
+        ValueError: If we are blindly passed a value that is None.
     """
+    if value is None:
+        raise ValueError("Cannot hash None.")
```

Comment:
added this because its a little more of an explicit error if someone passes None here.

Reply:
Very nice. Thanks. 👍